### ISSUE TITLE
Protect Main branch

### ISSUE THREAD

**Issue by jalpa015** [xcollab/sql#4]
@arun-esh 
How about protecting the main branch from PR merge and commits?

**Comment by arun-esh** [comment]
@jalpa015 Yeah! We must do it. I did some rules to the repo. Will test out soon.

**Comment by arun-esh** [comment]
It is done. Thanks
